shell/platform/linux/public/flutter_linux/fl_binary_messenger.h

```diff
@@ -187,6 +197,32 @@ GBytes* fl_binary_messenger_send_on_channel_finish(FlBinaryMessenger* messenger,
                                                    GAsyncResult* result,
                                                    GError** error);
 
+/**
+ * fl_binary_messenger_resize_channel:
+ * @binary_messenger: an #FlBinaryMessenger.
+ * @channel: channel to be resize.
+ * @new_size: the new size for the channel buffer.
+ *
+ * Sends a message to the control channel asking to resize a channel buffer.
+ */
+void fl_binary_messenger_resize_channel(FlBinaryMessenger* messenger,
+                                        const gchar* channel,
+                                        int new_size);
```

Comment:
Use int64_t as that's the size that's data type that's sent in the channel. I was also considering size_t, but it's probably better to match what is sent.